### ISSUE TITLE
[ci log] watchdog reset after 120 seconds

### ISSUE THREAD

**Issue by zixia** [Chatie/wechaty#195]
This issue is created for logging all the `timeout` problem.

---

https://travis-ci.org/wechaty/wechaty/jobs/190420734#L3913

```shell
SILL PuppetWebBrowser Browser.execute("return typeof WechatyBro === "undefined"")
SILL StateMonitor Browser:target() - open
SILL StateMonitor Browser:current() - open
SILL PuppetWebBridge execute()
SILL PuppetWebBrowser Browser.execute(" const callback = arguments[arguments.length - 1] const isAsync = (typeof  ... ")
SILL StateMonitor Browser:target() - open
SILL StateMonitor Browser:current() - open
SILL StateMonitor Puppet:target() - live
SILL StateMonitor Puppet:current() - dead
SILL StateMonitor Puppet:stable() is true
VERB PuppetWeb quit() state target(live) current(dead) stable(true)
SILL StateMonitor Puppet:current() - dead
SILL StateMonitor Puppet:inprocess() false
WARN PuppetWeb quit() is called on a `dead` browser. return directly.
  ✖ test › puppet-web › watchdog › timer should not get unexpected `error` event: watchDogReset() watchdog reset after 120 seconds, last feed:[HEARTBEAT:[feed to extend the dog life]], Error: watchDogReset() watchdog reset after 120 seconds, last feed:[HEARTBEAT:[feed to extend the dog life]]
    at PuppetWeb.<anonymous> (/home/travis/build/wechaty/wechaty/src/puppet-web/watchdog.ts:112:13)
    at Generator.next (<anonymous>)
    at /home/travis/build/wechaty/wechaty/dist/src/puppet-web/watchdog.js:7:65
    at __awaiter (/home/travis/build/wechaty/wechaty/dist/src/puppet-web/watchdog.js:3:12)
    at PuppetWeb.watchDogReset (/home/travis/build/wechaty/wechaty/dist/src/puppet-web/watchdog.js:94:12)
    at Timeout.watchDogTimer._ [as _onTimeout] (/home/travis/build/wechaty/wechaty/src/puppet-web/watchdog.ts:103:54)
    at ontimeout (timers.js:365:14)
    at tryOnTimeout (timers.js:237:5)
    at Timer.listOnTimeout (timers.js:207:5)
```

**Comment by zixia** [comment]
This bug is long time no see.

**Comment by Arvintian** [comment]
I met simlar problem when launch wechaty

2017/8/28 下午5:05:0709:05:07 INFO Wechaty v0.8.138 initializing...
2017/8/28 下午5:07:37Error: Error: watchDogReset() watchdog reset after 120 seconds, last feed:[HEARTBEAT:[inited]]
2017/8/28 下午5:07:37    at PuppetWeb.<anonymous> (/wechaty/dist/src/puppet-web/watchdog.js:100:19)
2017/8/28 下午5:07:37    at Generator.next (<anonymous>)
2017/8/28 下午5:07:37    at /wechaty/dist/src/puppet-web/watchdog.js:7:71
2017/8/28 下午5:07:37    at __awaiter (/wechaty/dist/src/puppet-web/watchdog.js:3:12)
2017/8/28 下午5:07:37    at PuppetWeb.watchDogReset (/wechaty/dist/src/puppet-web/watchdog.js:98:12)
2017/8/28 下午5:07:37    at Timeout.watchDogTimer._ [as _onTimeout] (/wechaty/dist/src/puppet-web/watchdog.js:92:56)
2017/8/28 下午5:07:37    at ontimeout (timers.js:386:14)
2017/8/28 下午5:07:37    at tryOnTimeout (timers.js:250:5)
2017/8/28 下午5:07:37    at Timer.listOnTimeout (timers.js:214:5)
2017/8/28 下午5:10:37Error: Error: watchDogReset() watchdog reset after 180 seconds, last feed:[HEARTBEAT:[onBrowserDead() set a timeout of 180 seconds to prevent unknown state change]]
2017/8/28 下午5:10:37    at PuppetWeb.<anonymous> (/wechaty/dist/src/puppet-web/watchdog.js:100:19)
2017/8/28 下午5:10:37    at Generator.next (<anonymous>)
2017/8/28 下午5:10:37    at /wechaty/dist/src/puppet-web/watchdog.js:7:71
2017/8/28 下午5:10:37    at __awaiter (/wechaty/dist/src/puppet-web/watchdog.js:3:12)
2017/8/28 下午5:10:37    at PuppetWeb.watchDogReset (/wechaty/dist/src/puppet-web/watchdog.js:98:12)
2017/8/28 下午5:10:37    at Timeout.watchDogTimer._ [as _onTimeout] (/wechaty/dist/src/puppet-web/watchdog.js:92:56)
2017/8/28 下午5:10:37    at ontimeout (timers.js:386:14)
2017/8/28 下午5:10:37    at tryOnTimeout (timers.js:250:5)
2017/8/28 下午5:10:37    at Timer.listOnTimeout (timers.js:214:5)

**Comment by zixia** [comment]
This issue should be fixed at v0.10 or above.
